lib/names/workloadendpoint_test.go

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package names_test
+
+import (
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	cerrors "github.com/projectcalico/libcalico-go/lib/errors"
+	"github.com/projectcalico/libcalico-go/lib/names"
+)
+
+func init() {
+	DescribeTable("WorkloadEndpoint name construction, fully qualified names",
+		func(ids names.WorkloadEndpointIdentifiers, name string, erroredField string) {
```

Comment:
nit: for table driven test, it's nice to have expected output variables named something like `expectedName` so it's easy to understand when someone is reading the test

Reply:
fixed.